Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -145,10 +147,19 @@ public NodejsProjectNode(NodejsProjectPackage package)
         }
 
         private void TryToAcquireTypings(IEnumerable<string> packages) {
+            bool isNewTypingsFolder = !Directory.Exists(Path.Combine(this.ProjectHome, "typings"));
```

Comment:
Shouldn't we also notify users in the case when we update an existing `typings` folder? That seems like more consistent behavior to me since it also results in an unexpected change to your local files.